src/sentry/relay/config/__init__.py

```diff
@@ -414,10 +414,13 @@ def _filter_option_to_config_setting(flt, setting):
         "d:transactions/measurements.fid@millisecond",
         "d:transactions/measurements.fp@millisecond",
         "d:transactions/measurements.frames_frozen@none",
+        "d:transactions/measurements.frames_frozen_rate@ratio",
         "d:transactions/measurements.frames_slow@none",
+        "d:transactions/measurements.frames_slow_rate@ratio",
         "d:transactions/measurements.frames_total@none",
         "d:transactions/measurements.stall_count@none",
         "d:transactions/measurements.stall_longest_time@millisecond",
+        "d:transactions/measurements.stall_percentage@percent",
```

Comment:
Unclear what the unit of this metric should be, see comment [here](https://github.com/getsentry/relay/pull/1373#discussion_r935240113).

Reply:
Based on the formula it looks like this is a [`fraction`](https://getsentry.github.io/relay/relay_metrics/enum.FractionUnit.html#variant.Ratio).